modmail/exts/meta.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from modmail.bot import ModmailBot
+from modmail.log import ModmailLogger
+from modmail.utils.cogs import CogMetadata
+
+log: ModmailLogger = logging.getLogger(__name__)
+COG_METADATA = CogMetadata()
+
+
+class Meta(commands.Cog):
+    """Meta commands to get info about the bot itself."""
+
+    def __init__(self, bot: ModmailBot):
+        self.bot = bot
+
+    @commands.command()
+    async def ping(self, ctx: Context) -> None:
+        """Check response time."""
```

Comment:
Better, thanks alec!
```suggestion
        """Ping the bot to see its latency and state."""
```